Sources/GeneralKit/Models/GeneralConfig.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright © 2021 Rosberry. All rights reserved.
+//
+
+public struct GeneralConfig: Codable, CustomStringConvertible {
+    public var version: String
+    public var templatesRepos: [String: String]
+    public var overrides: [String: String]
+
+    public init(version: String,
+                templatesRepos: [String: String],
+                pluginRepos: [String: String],
+                overrides: [String: String]) {
+        self.version = version
+        self.templatesRepos = templatesRepos
+        self.overrides = overrides
+    }
+
+    public func addingUnique<Value: Hashable>(_ value: Value, by keyPath: KeyPath<GeneralConfig, [Value]>) -> [Value] {
+        Array(Set(self[keyPath: keyPath] + [value]))
```

Comment:
```suggestion
        (self[keyPath: keyPath] + [value]).unique
```

Reply:
`unique` property does not exists. is it swift sequence function?

Reply:
My bad, this is from Base package